map/views.py

```diff
@@ -30,6 +32,38 @@ def handle_invalid_usage(error):
     return response
 
 
+def get_cascaders() -> List[str]:
+    return list(flask_redis.smembers("cascaders.users"))
+
+
+def find_cascader(cascaders: List[str], hash: str) -> Optional[str]:
```

Comment:
I know you’re not using it here, but it’s good practice not to override builtin methods (like `hash`)